.github/workflows/plugin-check.yml

```diff
@@ -0,0 +1,39 @@
+name: WordPress Plugin Checks
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    uses: 10up/classifai/.github/workflows/build-release-zip.yml@feature/plugin-check-action # TODO: Update branch before merge.
```

Comment:
Note this should be updated to `develop` before merging in (though I always forget and end up having to fix after)